tests_e2e/test_system.py

```diff
@@ -0,0 +1,177 @@
+            'description input': '/html/body/div/div/div[1]/div/form/div[2]/textarea',
+            'date input': '/html/body/div/div/div[1]/div/form/div[3]/input',
+            'new task_button': '/html/body/div/div/div[1]/div/form/div[4]/button'
+        }
+
+        self.card = {
+            'card name': '/html/body/div/div/div[3]/div[1]/div/div/h5',
+            'card date': '/html/body/div/div/div[3]/div[1]/div/div/p[2]'
+        }
+
+
+@pytest.fixture
+def test_app():
+
+    # # Create the system testing app using the 'system testing' config + the new board id as defined in the Config.py file
+    boardMaker = BoardMaker(config)
+    board_id = boardMaker.create_board()
+
+    # updating the system testing config with the newly created trello board
+    test_app = create_app('system testing', overwrite_board=board_id)
+
+    # start the app in its own thread.
+    thread = Thread(target=lambda: test_app.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield test_app
+
+    # Use the app to create a test_client that can be used in our tests.
+    thread.join(1)
+    boardMaker.delete_board(board_id)
+
+
+def test_task_journey(driver, test_app):
+    driver.get('http://localhost:5000/')
+    assert driver.title == 'To-Do App'
+
+    card = {
+        'todo': {
+            'card name': '/html/body/div/div/div[3]/div[1]/div/div/h5',
+            'card date': '/html/body/div/div/div[3]/div[1]/div/div/p[2]',
+            'card desc': '/html/body/div/div/div[3]/div[1]/div[1]/div/p[1]',
+            'card link': '/html/body/div/div/div[3]/div[1]/div[1]/div/a'
+        },
+        'done': {
+            'card name': '/html/body/div/div/div[3]/div[3]/div/div/h5',
+            'card date': '/html/body/div/div/div[3]/div[3]/div/div/p[2]',
+            'card desc': '/html/body/div/div/div[3]/div[3]/div[1]/div/p[1]',
+            'card link': '/html/body/div/div/div[3]/div[3]/div[1]/div/a'
+        },
+        'doing': {
+            'card name': '/html/body/div/div/div[3]/div[2]/div/div/h5',
+            'card date': '/html/body/div/div/div[3]/div[2]/div/div/p[2]',
+            'card desc': '/html/body/div/div/div[3]/div[2]/div[1]/div/p[1]',
+            'card link': '/html/body/div/div/div[3]/div[2]/div[1]/div/a'
+        }
+    }
+
+    entries = {
+        'task name': '/html/body/div/div/div[1]/div/form/div[1]/input',
+        'description input': '/html/body/div/div/div[1]/div/form/div[2]/textarea',
+        'date input': '/html/body/div/div/div[1]/div/form/div[3]/input',
+        'new task button': '/html/body/div/div/div[1]/div/form/div[4]/button'
+    }
+
+    link = {
+        'submit': '/html/body/div/div/div/div/form/div[5]/button[1]',
+        'delete': '/html/body/div/div/div/div/form/div[5]/button[2]'
+    }
+
+    status = {
+        'TODO': '//*[@id="radio1"]',
+        'DOING': '//*[@id="radio2"]',
+        'DONE': '//*[@id="radio3"]'
+    }
+
+    driver.find_element_by_xpath(
```

Comment:
This is quite a neat way to make the tests readable by having the xpaths be named, nice

Reply:
thanks